### PR TITLE
Adding link to devops stages

### PR DESCRIPTION
**Pull request by eugman**
It may not be clear to a Power BI user what a stage is. I tried to add a link to this documentation, but I may not have gotten the syntax right.
https://docs.microsoft.com/en-us/azure/devops/pipelines/get-started/key-pipelines-concepts?view=azure-devops#stage

Alternatively a sentence or two of definition might be sufficient.

### CODE REVIEW

**Comment by PRMerger20** [comment]
@eugman : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: powerbi-docs/create-reports/deployment-pipelines-get-started.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ ms.date: 02/14/2022
 
 This article walks you through the basic settings required for using deployment pipelines in Power BI service. It's recommended to read the [deployment pipelines introduction](deployment-pipelines-overview.md), before you proceed.
 
-In a deployment pipeline, one workspace is assigned to each stage. Before you start working with your pipeline in production, review the [capacity requirements](deployment-pipelines-troubleshooting.yml#what-type-of-capacity-can-i-assign-to-a-workspace-in-a-pipeline-) for the pipeline's workspaces.
+In a deployment pipeline, one workspace is assigned to each [stage](../../azure/devops/pipelines/get-started/key-pipelines-concepts#stage). Before you start working with your pipeline in production, review the [capacity requirements](deployment-pipelines-troubleshooting.yml#what-type-of-capacity-can-i-assign-to-a-workspace-in-a-pipeline-) for the pipeline's workspaces.
```
Comment (ShannonLeavitt) [inline powerbi-docs/create-reports/deployment-pipelines-get-started.md:19]:
```suggestion
In a deployment pipeline, one workspace is assigned to each [stage](../../azure/devops/pipelines/get-started/key-pipelines-concepts.md#stage). Before you start working with your pipeline in production, review the [capacity requirements](deployment-pipelines-troubleshooting.yml#what-type-of-capacity-can-i-assign-to-a-workspace-in-a-pipeline-) for the pipeline's workspaces.
```

**Comment by ShannonLeavitt** [comment]
#label:"aq-pr-triaged"

**Comment by PRMerger4** [comment]
@eugman : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by KesemSharabi** [comment]
Hi @eugman,

Thank you for your contribution!

However, your link isn't correct. [Deployment pipelines](https://docs.microsoft.com/power-bi/create-reports/deployment-pipelines-overview) and [Azure pipelines](https://docs.microsoft.com/azure/devops/pipelines/get-started/what-is-azure-pipelines?view=azure-devops) are two different features. Each feature has it's own definition of 'stage'. The deployment pipelines overview article (link above) explains this concept for deployment pipelines.  It is then further explained in the article you tried to add a link to. So overall, I think this concept is pretty well explained and doesn't need additional links.

Thanks,

Kesem

**Pull request closed without merging**